### PR TITLE
Added folder icon patch

### PR DESCRIPTION
**Pull request by hadicharara**
Hey,

So I added a small patch so that the symbolic folder icon for the bookmarks matches the one included in the Adwaita icon theme.
Here is the difference:
Screenshot from 2021-04-25 18-34-09
Screenshot from 2021-04-25 18-34-30

I'm not too familiar with firefox css, or css in general so there's probably a cleaner way to do what I did. Awesome theme btw!

### CODE REVIEW

File: theme/icons/folder-symbolic.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16"><path d="M1 1a1 1 0 00-1 1v11s-.014.459.23.947C.476 14.436 1.168 15 2 15h12s.459.014.947-.23C15.436 14.525 16 13.833 16 13V6H3v7H2V6s0-1 1-1h11v-.832A1 1 0 0013 3H8.414L6.707 1.293A1 1 0 006 1z" style="line-height:normal;font-variant-ligatures:normal;font-variant-position:normal;font-variant-caps:normal;font-variant-numeric:normal;font-variant-alternates:normal;font-feature-settings:normal;text-indent:0;text-align:start;text-decoration-line:none;text-decoration-style:solid;text-decoration-color:#000;text-transform:none;text-orientation:mixed;white-space:normal;shape-padding:0;isolation:auto;mix-blend-mode:normal;solid-color:#000;solid-opacity:1" color="#000" font-weight="400" font-family="sans-serif" overflow="visible" fill="#2e3436" fill-rule="evenodd"/></svg>
```
Comment (rafaelmardojai) [inline theme/icons/folder-symbolic.svg:1]: Please replace `fill="#2e3436"` with `fill="context-fill" fill-opacity="context-fill-opacity"` so we can recolor the icons with CSS.

File: theme/parts/icons.css [hunk 2]
```diff
@@ -338,3 +338,7 @@ menuitem[type="radio"][disabled="true"] .menu-iconic-icon {
 		list-style-image: var(--webextension-menupanel-image-light, inherit) !important;
 	}
 }
+
+.bookmark-item[container] {
```
Comment (rafaelmardojai) [inline theme/parts/icons.css:342]: You will also need to add this to the icons color statement from `icons.css#L6`.

**Review by rafaelmardojai — CHANGES_REQUESTED** [review]
Thanks, this way looks better!

File: theme/icons/folder-symbolic.svg (hunk 1 above)
Comment (hadicharara) [reply theme/icons/folder-symbolic.svg]: Done!

File: theme/parts/icons.css (hunk 2 above)
Comment (hadicharara) [reply theme/parts/icons.css:344]: And done!

**Review by rafaelmardojai — APPROVED** [review]

**Comment by rafaelmardojai** [comment]
Thanks!

**Pull request merged**